### ISSUE TITLE
Give each TableRow in search results a CSS class that corresponds to the column information.

### ISSUE THREAD

**Issue by RomanySaad** [Radarr/Radarr#6808]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

I would like to create custom Stylus styles for certain search result TableRows. (and TableCells, if possibel) For instance, highlight Bluray-2160p rows or make certain custom formats slightly faded. It would aid in finding the exact release I want at a glance.

### Describe the solution you'd like

Can we give each TableRow in the search results a CSS class that corresponds to the column information: source, indexer, flagged, etc. Also include classes for custom formats ("Custom-Scene" for a custom format named "Scene", for instance).

### Describe alternatives you've considered

I tried using Tampermonkey to create a jQuery solution since it can query cell text but couldn't easily find where to hijack the function/callback that occurs when the search completes. Turthfully, I didn't search for long.

### Anything else?

None, really. This would be custom CSS classes added on the TableRow (and, possibly, TableCell) to help facilitate custom style injection.

**Comment by ta264** [comment]
I don't think this is going to happen. Certainly not the custom format suggestion - the front-end classes are generated before any custom formats are set up.

Adding classes for all possible values for other items seems like dead weight code to me.

**Comment by RomanySaad** [comment]
I understand this isn’t exactly high priority. However, just to clarify, the custom css classes would be added by Radarr but it would be up to the user to style them accordingly via a browser extension such as stylus.
